ui/redux/actions/settings.js

```diff
@@ -42,6 +81,19 @@ export function doSetDaemonSetting(key, value) {
   };
 }
 
+export function doCacheCustomWalletServers(servers) {
+  return {
+    type: ACTIONS.WALLET_SERVERS_CACHED,
```

Comment:
See comment above, not a fan of this name.

You are actively saving the servers, they aren't cached at this point.